cmd/compute-domain-daemon/main.go

```diff
@@ -175,10 +182,22 @@ func run(ctx context.Context, cancel context.CancelFunc, flags *Flags) error {
 		computeDomainNamespace: flags.computeDomainNamespace,
 		nodeName:               flags.nodeName,
 		podIP:                  flags.podIP,
+		maxNodesPerIMEXDomain:  flags.maxNodesPerIMEXDomain,
 	}
 	klog.Infof("config: %v", config)
 
-	// Prepare IMEX daemon process manager (not invoking the process yet).
+	var dnsNameManager *DNSNameManager
+	if featuregates.Enabled(featuregates.IMEXDaemonsWithDNSNames) {
```

Comment:
So, would I enable that by running `helm install ... --set featureGates.IMEXDaemonsWithDNSNames=true`?

Reply:
yes